### DISCUSSION TITLE
What do you guys do with scrapes

### DISCUSSION THREAD

**Discussion by sintaxx** [UltimaHoarder/UltimaScraper#1290] [Q&A]
I'm curious what you guys are doing with your scrapes ? I have shit tons of content i've scraped and lately i've been dumping it all into a personal piwigo gallery which i spun up in docker but i can't figure out a way to make use of the metadata information that goes with the pictures. Are any of you doing anything similar ? Just looking for ideas to organize the mass amounts of content.

**Comment by UltimaHoarder** [comment]
If anyone answers truthfully, they're going to jail.

**Reply by sintaxx** [reply]
Hahaha, possibly. The magic is in the wording.

I'd just love for a way to automate dumping my scrapes into some sort of gallery software.

**Reply by americanseeder1865** [reply]
You could probably do this with Docker Bind Mounts, Chrontab (to autmate execution of the scrapper and tell your gallery software to index if applicable), and properly configuring the .settings file.

**Comment by cjemorton** [comment]
Lets say hypothetically: Highspeed Connection while at one location; at another location there is basically no internet, only spotty and extremely limited overpriced options. So; being able to dump the subs into a folder to later be able to actually take a look at. Well that is a "legit" use hypothetically.

**Reply by fg4jerem** [reply]
Yes, and that's exactly what I use it for.

I can't stand OF's platform, it's laggy, slow, often down. I prefer to "consume" the content locally after it's been downloaded.
I wish there was better access to the meta-data tho. Currently it's hidden in a .db file and I miss all the wonderful stories some of my creators post.

**Reply by cjemorton** [reply]
You can use a SQLite DB Browser (Google It) to open the metadata file. This is text only as your simply viewing your locally scraped database.

**Reply by Luke-L** [reply]
> Yes, and that's exactly what I use it for.
> 
> I can't stand OF's platform, it's laggy, slow, often down. I prefer to "consume" the content locally after it's been downloaded.
> I wish there was better access to the meta-data tho. Currently it's hidden in a .db file and I miss all the wonderful stories some of my creators post.

same. their media browser on each users profile is fucking abysmal.

**Comment by maxcom99** [comment]
Hypothetically, there is a project called "stash" on github ...

**Reply by cjemorton** [reply]
Hypothetically, this has potential.

**Reply by sintaxx** [reply]
this is amazing! is there anyone on here with the ability to create a scraper/plugin so Stash can read metadata/etc from this projects scraper to organize the content better ?

**Comment by LunarPenguin** [comment]
I would like to have access to content after my subscription to a model ends, because i don't normally sub to a model for longer than a month. there are some exceptions though.
